### PR TITLE
Terminus fixes fixes

### PR DESCRIPTION
**Pull request by alexfornuto**
## Effect
PR includes the following changes:
- Removes Terminus as project dependency instructions.
- Chmod likes a little x.

## Remaining Work
- [x] Technical review

## Post Launch
**Do not remove** - To be completed by the docs team upon merge:
- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

File: source/content/terminus/install.md [hunk 1]
```diff
@@ -52,7 +51,7 @@ There are several ways to install Terminus, depending on your use case:
 1. Make the Terminus file executable. The example below assumes the same installation path as above:
 
   ```bash{promptUser: user}
-  chmod +X ~/.bin/terminus
+  chmod +x ~/.bin/terminus
```
Comment (greg-1-anderson) [inline source/content/terminus/install.md]: Also the text says `$HOME/bin` so maybe this should be `$HOME/bin/terminus` or at least `~/bin/terminus`

Comment (alexfornuto) [reply source/content/terminus/install.md]: Why?

Comment (greg-1-anderson) [reply source/content/terminus/install.md]: Invisible `.bin` is unusual, but I'm not going to stop you; the reader can reverse it if they have a different preference. As long as it's consistent.

**Comment by greg-1-anderson** [comment]
While we are here, could we also make this part a bit more perscriptive:
```
Install the most recent release of Terminus with the following command within a directory where you have permission to write files. If in doubt, you can create a `terminus` directory in your `$HOME` and install there:
```
Folks have run into problem by using the Terminus installer in $HOME and then installing something else in $HOME. Maybe we coudl add `mkdir terminus` and `cd terminus` to the instructions as explicit steps instead of a suggestion?

File: source/content/terminus/install.md (hunk 1 above)
Comment (alexfornuto) [reply source/content/terminus/install.md]: Unusual to who? Are there conventions on this? What's the latest study show?

`/me` uses .bin, fwiw.

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5542 | Difference |
| --- | --- | --- | --- |
| Performance Score | `54` | `41` | `-13` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5542` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18706/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18706/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5542 | Difference |
| --- | --- | --- | --- |
| Performance Score | `56.00000000000001` | `47` | `-9` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5542` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18712/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18712/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

File: source/content/terminus/install.md (hunk 1 above)
Comment (greg-1-anderson) [reply source/content/terminus/install.md]: Unusual as in I have seen `$HOME/bin` N times, for a significantly large value of N, whereas I have seen `$HOME/.bin` one time. Also, if you google for `/home/.bin` you will find quite a few references to `bin` (still the minority in search results that are primarily about trash cans), but none to `.bin` in the first ten pages.

However, there is no RFC, so you may do as you like here.

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5542 | Difference |
| --- | --- | --- | --- |
| Performance Score | `51` | `49` | `-2` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5542` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18725/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18725/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Pull request merged**